Sources/Target/POSIX/ELFProcess.cpp

```diff
@@ -173,7 +173,7 @@ EnumerateLinkMap(ELFProcess *process, Address addressToDPtr,
   // FIXME: main executable detection is disabled because this causes some
   // issues when using the mozzloader. LLDB seems to be fine with us not
   // marking any object as being the main executable anyway.
-  bool isMain = false;
+  bool isMain = true;
```

Comment:
The commit message says that this is only for android, but I don't see any platform-specific code?